libavcodec/qsvenc.c

```diff
@@ -988,6 +995,15 @@ static int init_video_param(AVCodecContext *avctx, QSVEncContext *q)
 #if QSV_VERSION_ATLEAST(1, 16)
             if (q->int_ref_cycle_dist >= 0)
                 q->extco3.IntRefCycleDist = q->int_ref_cycle_dist;
+#endif
+#if QSV_VERSION_ATLEAST(1, 23)
+            q->extco3.LowDelayBRC = q->low_delay_brc ? MFX_CODINGOPTION_ON : MFX_CODINGOPTION_UNKNOWN;
```

Comment:
Will the SDK choose LowDelayBRC automatically for MFX_CODINGOPTION_UNKNOWN ? LowDelayBRC is a tri-state option in the SDK, it would be better to turn low_delay_brc into a tri-state option too.

Reply:
Agree to change it to tri-state. Will update it.
In current SDK, it turns off LowDelayBRC for MFX_CODINGOPTION_UNKNOWN, as the default option.